ext/standard/tests/general_functions/ini_set_types.phpt

```diff
@@ -0,0 +1,29 @@
+--TEST--
+ini_set() accepts non-strings under strict_types
+--FILE--
+<?php
+declare(strict_types=1);
+
+ini_set('docref_root', null);
+var_dump(ini_get('docref_root'));
+ini_set('html_errors', true);
+var_dump(ini_get('html_errors'));
+ini_set('html_errors', false);
+var_dump(ini_get('html_errors'));
+ini_set('precision', 6);
+var_dump(ini_get('precision'));
+// Are there any float options?
```

Comment:
"opcache.jit_prof_threshold"  and "session.upload_progress.min_freq" seem to be the only ones looking at https://heap.space/s?refs=OnUpdateReal&project=php-src

Reply:
Heh, I search for OnUpdateDouble and OnUpdatFloat ... didn't expect it to be called OnUpdateReal!

Reply:
Both are part of optional exts and session.upload_progress.min_freq is PERDIR :/